src/main/java/org/catrobat/jira/timesheet/helper/CsvTimesheetExporter.java

```diff
@@ -50,8 +50,6 @@ public String getTimesheetCsvData(Timesheet timesheet) {
                 "Lecture" + DELIMITER);
 
         sb.append(timesheet.getUserKey()).append(DELIMITER);
-        sb.append(Integer.toString(timesheet.getHoursPracticeCompleted())).append(DELIMITER);
```

Comment:
This removes the column for the timesheets, however there is still the header column saying `Username;Practical Hours;Hours Done;` (header now differs from data rows).

Reply:
Done